kratos/includes/mapping_variables.h

```diff
@@ -0,0 +1,40 @@
+//    |  /           |
+//    ' /   __| _` | __|  _ \   __|
+//    . \  |   (   | |   (   |\__ \.
+//   _|\_\_|  \__,_|\__|\___/ ____/
+//                   Multi-Physics
+//
+//  License:		 BSD License
+//					 Kratos default license: kratos/license.txt
+//
+//  Main authors:    Vicente Mataix Ferrándiz
```

Comment:
A minor comment, but please refrain from using accented characters (or non-ascii in general) in source code files, since it tends to produce errors when changing between windows and linux.